services/minespace-web/src/components/common/SteppedForm.js

```diff
@@ -48,7 +48,8 @@ const SteppedForm = (props) => {
 
   const handleNextClick = (evt, tab) => {
     evt.preventDefault();
-    handleTabClick(tab);
+    handleSaveData(null, tab);
```

Comment:
Hmmm, we'll need to think how we want to handle saves on tab changes.  I think this is going to trigger a double save for 'Basic Information' and 'EOR' since those saves are triggered within `SteppedForm`

Reply:
Good point! My opinion would be that saves should only happen when you hit "next" and not when you switch tabs, if anything we could reset the form with a "are you sure you want to discard your current changes?" prompt if you switch tabs without hitting next. 

Will check in with Roop/Raj on this and update it across the board

Reply:
Yeah, that would work for me!